docs/event-bus/03-03-service-programming-model.md

```diff
@@ -23,10 +23,10 @@ The following HTTP Headers provide information about the Event metadata.
 
 |Header| Description|
 |------|--------|
-| **ce-event-id** | Business Event's ID delivered to the microservice. |
-| **ce-event-time** | Business Event's time delivered to the microservice. |
-| **ce-event-type** | Business Event's type delivered to the microservice. |
-| **ce-event-type-version** | Business Event's version delivered to the microservice. |
+| **ce-id** | Business Event's ID delivered to the microservice. |
```

Comment:
The headers [here](https://github.com/kyma-project/kyma/pull/4585/files#r296583732) should be added as well.

Reply:
Done!